src/main/java/spoon/support/reflect/declaration/CtElementImpl.java

```diff
@@ -127,8 +127,8 @@ public boolean equals(Object o) {
 			return false;
 		String current = getSignature();
 		String other = ((CtElement) o).getSignature();
-		if (current.length() <= 0 || other.length() <= 0)
-			throw new CtUncomparableException("Unable to compare elements");
+		//if (current.length() <= 0 || other.length() <= 0)
+		//	throw new CtUncomparableException("Unable to compare elements");
```

Comment:
Yep, thanks. Because you wrote a test case which will fail if we uncomment these lines. So, they are no more necessary.